src/components/markdown/renderers.tsx

```diff
@@ -71,3 +74,70 @@ export const InlineCodeRenderer: React.FC<ReactMarkdownCodeRendererProps> = (
 ) => {
   return <InlineCode>{props.value}</InlineCode>;
 };
+
+/*
+ * Image
+ *
+ */
+const StyledImage = styled.img`
+  width: 100%;
+`;
+
+const isCloudinaryUrl = (src: string): boolean => {
```

Comment:
this is kind of cluttering the renderer file. I think everything around the cloudinary would fit in a separate file very good?

Reply:
good idea, makes it also simpler for testing 👍